app/src/main/java/com/android/halanvendordiscovery/data/details/model/response/LocationRemoteModel.kt

```diff
@@ -0,0 +1,6 @@
+package com.android.halanvendordiscovery.data.details.model.response
```

Comment:
it wasn't clear for me why Islam created the (response) package, I discussed it with him and I understood that this is to separate the response models from request models